text_generation/causal_lm/cpp/README.md

```diff
@@ -36,6 +36,18 @@ The program loads a tokenizer, a detokenizer and a model (`.xml` and `.bin`) to
 
 The program loads a tokenizer, a detokenizer and a model (`.xml` and `.bin`) to OpenVINO. A prompt is tokenized and passed to the model. The model predicts a distribution over the next tokens and group beam search samples from that distribution to explore possible sequesnses. The result is converted to chars and printed.
 
+### speculative_sampling_lm
```

Comment:
we can also provide a reference to https://huggingface.co/blog/assisted-generation as reference article

Reply:
cited this link and original papers